Classes/Command/JobCommandController.php

```diff
@@ -74,6 +75,11 @@ public function workCommand($queue, $exitAfter = null, $limit = null, $verbose =
             }
             $this->outputLine('...');
         }
+        if (function_exists('pcntl_signal')) {
+            pcntl_signal(SIGINT, static function () {
+                throw new InterruptException('Interrupted by SIGINT');
```

Comment:
```suggestion
                throw new InterruptException('Interrupted by SIGINT', 1602072222);
```
I forgot in my first round: This is missing a unique exception number. We usually add the unix timestamp to the `throw` calls in order to be able to trace them back to the calling side more easily